RecyclingGame2D/Assets/Scripts/OvercookedScripts/SpriteChanger.cs

```diff
@@ -1,11 +1,14 @@
 using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
+using UnityEngine.UI;
+using TMPro;
 
 public class SpriteChanger : MonoBehaviour
 {
     private SpriteRenderer spriteRenderer;
     public Sprite Sprite1,Sprite2,Sprite3,Sprite4;
```

Comment:
Consider using a list for this, makes it easy to dynamically add sprites within editor. Will also clean up the manual code below, because you can just get sprite by index, and get there name thereafter.